README.md

```diff
@@ -1,39 +1,126 @@
+| TPM Individual Enrollment   | :heavy_minus_sign: | :heavy_minus_sign: | :heavy_minus_sign: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [Trusted Platform Module](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#trusted-platform-module-tpm).  Please review the [samples](./azure-iot-device/samples/) folder and this [quickstart](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-tpm-python)on how to create a device client.    |
+| X.509 Individual Enrollment | :heavy_check_mark: | :heavy_check_mark: | :heavy_minus_sign: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [X.509 root certificate](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#root-certificate).  Please review the [samples](./azure-iot-device/samples/) folder and this [quickstart](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-x509-python) on how to create a device client.              |
+| X.509 Enrollment Group      | :heavy_check_mark: | :heavy_check_mark: | :heavy_minus_sign: | This SDK supports connecting your device to the Device Provisioning Service via [group enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [X.509 leaf certificate](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#leaf-certificate)).  Please review the [samples](./azure-iot-device/samples/) folder on how to create a device client.                                                                                                                                  |
+| Symmetric Key Enrollment    | :heavy_check_mark: | :heavy_check_mark: | :heavy_minus_sign: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [Symmetric key attestation](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-symmetric-key-attestation)).  Please review the [samples](./azure-iot-device/samples/) folder on how to create a device client.                                                                                                                          |
+
+### IoTHub Service Library
```

Comment:
>IoTHub Service Library [](start = 4, length = 22)

As above, I think this should probably contain the package name with a link in brackets, i.e. `IoTHub Service Library (azure-iot-hub)`